resources/js/Components/Table.vue

```diff
@@ -0,0 +1,229 @@
+<template>
+  <table
+    ref="Table"
+    :class="[
+      'wikit',
+      'wikit-Table',
+      `wikit-Table--linear-${breakpoint}`
+    ]"
+  >
+    <slot />
+  </table>
+</template>
+
+<script setup lang="ts">
+import { PropType, computed, ref } from 'vue';
+import { Breakpoint, validateBreakpoint } from '../types/Breakpoint';
+
+/**
+ * Tables display categorical information organized across rows and columns in
+ * order to facilitate the comparative analysis of data.
+ *
+ * The WiKit table component provides a wrapper around the common HTML table
```

Comment:
Let's not forget to remove any mentions to Wikit ... :grimacing:

Reply:
ha! nice catch